respect-lib-shared/src/commonMain/kotlin/world/respect/shared/domain/report/model/ReportSeries.kt

```diff
@@ -0,0 +1,96 @@
+package world.respect.shared.domain.report.model
+
+import kotlinx.datetime.DatePeriod
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import org.jetbrains.compose.resources.StringResource
+import world.respect.shared.generated.resources.Res
+import world.respect.shared.generated.resources.*
+
+@Serializable
+data class ReportSeries(
+
+    val reportSeriesUid: Int = 0,
+
+    val reportSeriesTitle: String = "Series 1",
+
+    @Contextual
```

Comment:
It's working/compiling - and you don't know why adding @Contextual made the problem go away. BAD.